lib/PuppeteerSharp/Puppeteer.cs

```diff
@@ -32,21 +33,22 @@ public class Puppeteer
         /// </summary>
         /// <param name="options">Options for launching Chrome</param>
         /// <param name="chromiumRevision">The revision of Chrome to launch.</param>
+        /// <param name="loggerFactory">The logger factory</param>
         /// <returns>A connected browser.</returns>
         /// <remarks>
         /// See <a href="https://www.howtogeek.com/202825/what%E2%80%99s-the-difference-between-chromium-and-chrome/">this article</a>
         /// for a description of the differences between Chromium and Chrome.
         /// <a href="https://chromium.googlesource.com/chromium/src/+/lkcr/docs/chromium_browser_vs_google_chrome.md">This article</a> describes some differences for Linux users.
         /// </remarks>
-        public static Task<Browser> LaunchAsync(LaunchOptions options, int chromiumRevision)
-            => new Launcher().LaunchAsync(options, chromiumRevision);
+        public static Task<Browser> LaunchAsync(LaunchOptions options, int chromiumRevision, ILoggerFactory loggerFactory = null) 
+            => new Launcher(loggerFactory).LaunchAsync(options, chromiumRevision);
```

Comment:
Ok. So if you use a DI you wouldn't use the `Puppeteer` class. You'd declare a `Launcher` in your constructor.